IEEE-PSH/hackpsh-web
# Home layouts countdown

# Description
- Implemented working countdown
- Added TRPC route to retrieve time remaining

If the event time is upcoming, render countdown until.
If the event is ongoing, render countdown during.
If the event is complete, render event is complete.

# Test Guidelines
- Go to Home and check whether countdown functions as intended.

(To fully test this feature, change the start time in the TRPC route to test transitions between type of countdown)
